src/sentry/replays/lib/consumer.py

```diff
@@ -22,12 +22,16 @@ def submit(self, message: Message[TPayload]) -> None:
 
         try:
             self.__next_step.submit(message)
+        except MessageRejected:
+            raise
         except Exception:
             self.__logger.exception(self.__exception_message)
 
     def poll(self) -> None:
         try:
             self.__next_step.poll()
+        except MessageRejected:
+            raise
```

Comment:
I think this one is not needed, as Arroyo only handles MessageRejected being raised on submit.